### ISSUE TITLE
Pasrsing files (FormData)

### ISSUE THREAD

**Issue by aberba** [tchaloupka/httparsed#1]
Do you plan to support parsing Content-Disposition headers?

**Comment by tchaloupka** [comment]
Yes it should probably be part of the library similar to ie helper for processing chunked encoding to be usable in at least most common cases without the need to write own each time.

Slicing through `multipart/form-data` should be definitely there, parsing form parts headers too (`parseHeaders` should handle this already so we just need to use it in a form-data parser)

**Comment by zoujiaqing** [comment]
Hi @tchaloupka !

`httparsed` is a very useful and high quality library! I have integrated it into my new project, [Archttp](https://github.com/kerisy/archttp), and added the body parsing part.

However, in the process of use, some code had to be modified to integrate.

In the future version I will make 'httparsed' an external dependency.

Thanks again!
